### PR TITLE
[CARBONDATA-2127] Documentation for Hive Standard Partition

### PR DESCRIPTION
**Pull request by sgururajshetty**
Documentation for Hive Standard Partition

### CODE REVIEW

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -764,6 +765,107 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
...
+
+  This command gets the Hive partition information of the table
+
+  ```
+  SHOW PARTITIONS [db_name.]table_name
+  ```
+
+### Drop Partition
+
+  This command drops the specified Hive paritionOnly.
```
Comment (ravipesala) [inline docs/data-management-on-carbondata.md]: typo parition only

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**